css/style.css

```diff
@@ -0,0 +1,51 @@
+.graybackground{
+  background-color: rgb(234, 229, 229);
+}
+main{
+  display: block;
+  float:right;
+}
+#main-doc{
+  line-height: 150%;
+  position: absolute;
+  margin-left: 310px;
+}
+
+#navbar {
+  position: fixed;
+  min-width: 290px;
+  top: 0px;
+  left: 0px;
+  width: 300px;
+  height: 100%;
+  background-color:blanchedalmond;
+  border-right: dashed;
+  border-color: rgba(0, 22, 22, 0.4);
+}
+nav{
+  display: block;
+  position: fixed;
+  float: left;
+}
+@media only screen and (max-width: 650px){
+#navbar {
+  background-color: white;
+  position: absolute;
+  top: 0;
+  padding: 0;
+  margin: 0;
+  width: 100%;
+  max-height: 275px;
+  border: none;
+  z-index: 1;
+  border-bottom: 2px solid;
+}
+}
+@media only screen and (max-width: 650px)
+{
```

Comment:
Please make sure to fix the indentation problems in these lines. Remember for best practices, make sure to use two spaces as the standard indentation.